azerothcore/azerothcore-wotlk
# fix(DB/creature): adds 8 spawns for Giant Webwood Spider

## Changes Proposed:
-  Adds 8 spawns for Giant Webwood Spiders in Teldrassil, as there are only 4 spawns currently on AC and sources have many more

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8114

## SOURCE:

https://tbc.wowhead.com/npc=2001/giant-webwood-spider#comments
https://www.youtube.com/watch?v=lsbV8qmN02U
https://www.youtube.com/watch?v=sXeikPALlSE&t=512s

## Tests Performed:

- Ran SQL, no errors
- checked db and in-game

## How to Test the Changes:


1. `.go c 209189` and others
2. check their position and that they are moving
3. check with the videos if you like

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
